torch/_torch_docs.py

```diff
@@ -9836,6 +9836,35 @@ def merge_dicts(*dicts):
 `1` appears `n2` times, `2` appears `n3` times, etc.
 """.format(**common_args))
 
+add_docstr(torch.tile,
+           r"""
+tile(input, reps) -> Tensor
+
+Construct a Tensor by repeating the elements of the input tensor the number of times given by reps.
+
+.. note::
+
+    This is similar to ``numpy.tile``.
+
+Args:
+    input (Tensor): A tensor whose elements to repeat.
+    reps (tuple): The number of repetitions per dimension.
+
+Returns:
```

Comment:
No need for a Returns: here. We typically only use it for complicated return values, like tuples of tensors.